api/configs/personal_config_example.py

```diff
@@ -0,0 +1,11 @@
+from configs.configs import Config
+
+'''
+README:
+- Name your config 'personal.py'
+- Name your class 'PersonalConfig'. And inherit from the Config class i.e. PersonalConfig(Config).
+'''
+class PersonalConfigExample(Config):
```

Comment:
into version control, like if it's in a commit? i'm assuming if it comes up in a PR, someone should flag that then move it into a vault or smth

Reply:
By this time, it's too late. The PR is made up of commits.

Reply:
i don't think i'm understanding you're statement correctly. can you give me an example? is the goal to not merge credentials to main? that's what I'm interpreting from "accidentally put secrets into version control". if creds are in the commits, remove credentials, squash commits, then there shouldn't be any creds in main.

Reply:
You're right, there is a procedure to remove the secrets. Let's minimize the opportunity for secrets to be introduced in the first place, though. Future volunteers might miss seeing it, not know a credential shouldn't be put here, or not know the correct procedures to remove it or change it. First thought is to add a comment to tell future devs where to put secrets.

Reply:
definitely. i'll add to the readme as well as comments in the implementation 👌

Reply:
I now realized my initial code review comment that I put on this file should have been made in the `config.py` file where there is a code comment about `ProductionConfig`. The reason is because the instructions in the code comments in _this_ file (`personal_config_example.py`) instructs the developer to create `personal.py` which is already git ignore as part of this PR; thus, there won't/shouldn't be an issue of committing secrets put into the `personal.py` file. But, the concern still exists when applied to the `ProductionConfig` or `Config` classes. This PR has been updated in `config.py` to instruct the developer not to add secrets to reflect this conversion.

I hope that helps those who are following along.